### PR TITLE
Add dashboard role handler

### PR DESCRIPTION
**Pull request by imobachgs**
Add a system role handler for the dashboard role. Moreover, does some refactoring to the system roles handlers mechanism.

See https://trello.com/c/PJqM8x0T for details.

### CODE REVIEW

**Comment by imobachgs** [comment]
Thanks for the review! :)

File: src/lib/installation/system_role_handlers_runner.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, contact SUSE.
+#
+# To contact SUSE about this file by physical or electronic mail, you may find
+# current contact information at www.suse.com.
+# ------------------------------------------------------------------------------
+
+require "yast"
+require "installation/cfa/salt"
```
Comment (jreidinger) [inline src/lib/installation/system_role_handlers_runner.rb]: why it is needed here??? It is for worker role only, so it should be in that file.

Comment (imobachgs) [inline src/lib/installation/system_role_handlers_runner.rb]: You're right. I'll fix it (without version bumping) and anyway I'll remove it when merging into master.

**Comment by kobliha** [comment]
@imobachgs Just a side note: This really looks like something not generic enough, something that probably belongs to a CaaSP-specific package. But it's just an opinion to consider...

**Comment by imobachgs** [comment]
@kobliha Yes, it sounds good: we could move those roles handlers to a `yast-caasp` package. And we should think also about more code that could be included in that specific package.

**Pull request merged**